src/ui/notebook/NotebookHeader.tsx

```diff
@@ -31,8 +40,16 @@ export default function NotebookHeader({ kernelName }: Props) {
             <RefreshCwIcon size={16} />
           </button>
 
-          <button className="mx-2 w-60 rounded border border-gray-200 py-[3px] text-xs text-gray-900 transition-all hover:border-gray-400 hover:bg-gray-100 active:scale-105">
+          <button className="mx-2 flex w-60 items-center justify-center rounded border border-gray-200 py-[3px] text-xs text-gray-900 transition-all hover:border-gray-400 hover:bg-gray-100 active:scale-105">
+            <DotIcon
+              className={clsx(
+                "w-6",
+                kernelId === "" ? "text-orange-500" : "text-green-500",
+              )}
+            />
             {kernelName}
+            <div className="w-6" />{" "}
+            {/* Same width as the icon to make sure kernel name is center aligned */}
```

Comment:
I think it looks better when the whole thing is centered honestly. Also if you just want a circle, to avoid vertically expanding the box because the Dot is too small within its bounds we could just do a `<div w-6 h-6 rounded-full bg-orange-500`